lib/code_metrics/stats_directories.rb

```diff
@@ -28,6 +28,9 @@ def initialize
       @root = path_prefix
       @app_directories = default_app_directories
       @test_directories = default_test_directories
+      @test_directories.each do |pair|
+        CodeMetrics::Statistics::TEST_TYPES << pair.first
+      end
```

Comment:
I thought about that -- I guess it depends on whether or not you expect `default_test_directories` to be public and/or called more than once, so I put it in `initialize` so that it was only evaluated once (also hopefully ensuring we don't get dupes in TEST_TYPES).